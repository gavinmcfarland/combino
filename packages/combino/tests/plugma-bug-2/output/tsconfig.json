{
	"files": [],
	"references": [{ "path": "src/main/tsconfig.json" }, { "path": "src/ui/tsconfig.json" }]
}
