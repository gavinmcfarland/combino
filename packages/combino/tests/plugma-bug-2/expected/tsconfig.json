{
	"files": [],
	"references": [{ "path": "src/main/tsconfig.json" }]
}
